bigtable/tests/unit/test_row.py

```diff
@@ -389,40 +415,39 @@ class ErrorUnavailable(grpc.RpcError, grpc.Call):
         result = _retry_commit_exception(ValueError)
         self.assertNotEqual(result, True)
 
-    def test_commit_too_many_mutations(self):
-        from google.cloud._testing import _Monkey
-        from google.cloud.bigtable import row as MUT
-
-        row_key = b'row_key'
-        table = object()
-        row = self._make_one(row_key, table)
-        row._pb_mutations = [1, 2, 3]
-        num_mutations = len(row._pb_mutations)
-        with _Monkey(MUT, MAX_MUTATIONS=num_mutations - 1):
-            with self.assertRaises(ValueError):
-                row.commit()
```

Comment:
Why is this test gone?  Don't we need to check the case that there are too many mutations?

Reply:
@tseaver on our last meeting with Mr. Solomon. We are shifting this Max Mutations check on RowMutation class which is on mutation.py. That is why I removed this from test_row.py

Reply:
@aneepct: this test needs to be somewhere accessible to DirectRow.  Why does it have to be removed from here?